ctwrap/simulation.py

```diff
@@ -685,13 +685,13 @@ def worker(tasks_to_accomplish, tasks_that_are_done, module: str, lock,
             obj.run(task, config, **kwargs)
             with lock:
                 obj._save(task=task)
-
             msg = 'case `{}` completed by {}'.format(task, this)
             tasks_that_are_done.put(msg)
 
     if verbosity > 1:
         print("Appending metadata")
 
```

Comment:
Isn’t this only necessary if the object hasn’t been created previously? I’d suggest to set obj to None before the while loop and only do this if it’s still None herr